.github/workflows/ci.yml

```
@@ -409,6 +409,18 @@ jobs:
       id-token: write # for docker/login to login to NGINX registry
     secrets: inherit
 
+  openshift-certification:
+    name: OpenShift Certification
+    needs: [build-oss, build-plus, build-operator]
+    if: ${{ inputs.dry_run || false }}
+    uses: ./.github/workflows/openshift-certification.yml
+    with:
+      runner: ${{ github.repository_owner == 'nginx' && (inputs.is_production_release || (github.event_name == 'push' && github.ref == 'refs/heads/main')) && 'ubuntu-24.04-amd64' || 'ubuntu-24.04' }}
```

```suggestion
      runner: ubuntu-24.04
```

This doesn't need to run on the internal runners